examples/gallery/maps/country_polygons.py

```diff
@@ -0,0 +1,71 @@
+"""
+Highlight country and continent polygons
+----------------------------------------
+The :meth:`pygmt.Figure.coast` method can
+highlight country polygons via the ``dcw``
+parameter. It accepts the country code, and
+can draw its borders and add a color to its landmass.
+"""
+
+# sphinx_gallery_thumbnail_number = 2
+
+
+import pygmt
+
+fig = pygmt.Figure()
+
+fig.basemap(
+    region=[-12, 32, 34, 62],
+    projection="M6c",
+    frame=True,
+)
+
+fig.coast(
+    land="gray",
+    water="white",
+    dcw=[
+        # Great Britain with seagrean land
+        "GB+gseagreen",
+        # Italy with a red border
+        "IT+p0.75p,red3",
+        # Spain with a magenta dashed border
+        "ES+p0.75p,magenta4,-",
+        # Romania with a black dotted border
+        "RO+p1p,black,.",
+        # Germany with orange land and a blue border
+        "DE+gorange+p1p,dodgerblue4",
+    ],
+)
+fig.show()
```

Comment:
Just for consistency. I am also fine with the other way around, i.e., removing the corresponding blank line in the second example.
```suggestion
)

fig.show()
```